bin.src/generateInstCat.py

```diff
@@ -1,17 +1,24 @@
 #!/usr/bin/env python
 import argparse
 import warnings
+import os
 from astropy._erfa import ErfaWarning
 
 parser = argparse.ArgumentParser(description='Instance catalog generator')
 parser.add_argument('--db', type=str,
                     help='path to the OpSim database to query')
 parser.add_argument('--agn_db_name', type=str,
                     help='File of AGN parameters generated by create_agn_db.py')
+parser.add_argument('--host_image_dir', type=str,
+                    help='Location of FITS stamps of lensed host images produced by generate_lensed_hosts_***.py',
+                    default=os.path.join(os.environ['SIMS_GCRCATSIMINTERFACE_DIR'], 'data', 'outputs'))
+parser.add_argument('--host_data_dir', type=str,
+                    help='Name of csv file of lensed host data created by the sprinkler.', 
+                    default=os.path.join(os.environ['TWINKLES_DIR'],'data'))
```

Comment:
Should `host_data_dir` default to `os.path.join(os.environ['SIMS_GCRCATSIMINTERFACE_DIR'], 'data')`?

That seems to be where the `generate_lensed_hosts_*.py` scripts write to.